catalogue-access-service/data-dump/README.md

```diff
@@ -0,0 +1,82 @@
+# Resource server data (Elasticsearch Dump)
+
+Due to large size of datasets, its uploaded in drive ( and not in github/git LFS)
+
+Please find the datasets at the link : https://drive.google.com/drive/folders/1mbllMFJJw146O5TJQILUXURj80eqh7Wp
+
+The link contains the following sets:
+1)Providers : cat_provider.json
+2)Resource Servers : cat_resource_servers.json
+3)Resource Groups : cat_resource_groups.json
+4)Resource Items : cat_resource_items.json
+
+# Create ES Index dump
+
+Stack yaml (deploy in same swarm as es)-
```

Comment:
This is dump technique use to dump surat-itms, pune-env-aqm indices to csv format. Add the instructions on how you took different dumps of cat using elasticdump